components/gfx_traits/lib.rs

```diff
@@ -28,6 +28,12 @@ impl Epoch {
     }
 }
 
+impl Into<webrender_api::Epoch> for Epoch {
+    fn into(self) -> webrender_api::Epoch {
+        webrender_api::Epoch(self.0)
+    }
+}
+
```

Comment:
I think we just had one comment about using `From` instead of `Into`. So, I am approving this PR as looks good to me.

Reply:
Looks like [rust 1.41 has relaxed the coherence rules](https://blog.rust-lang.org/2020/01/30/Rust-1.41.0.html#relaxed-restrictions-when-implementing-traits) so that `From<Type>` is no longer considered a foreign trait if `Type` is owned by current trait, allowing us to implement `From<Type>` for foreign types like `webrender_api::Epoch`.

I'll raise a PR to make this change.

Reply:
Oh, wow! This is news to me. Thank you.